### PR TITLE
Adding byYearDay in event

### PR DESCRIPTION
**Pull request by edtexcodes**
Adding byYearDay for repeating frequency in generating ical file.

### CODE REVIEW

**Comment by evelynhathaway** [comment]
It looks like you deleted [`package-lock.json`](https://docs.npmjs.com/files/package-lock.json). Which provides a reproducable dependency tree for reliable debugging and testing. Was this an accident or was there a dependency that needs a bump in the lockfile?

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ calendar feeds.
 
 ## Installation
 
-	npm install ical-generator
+	npm install ical-generator-edtex
```
Comment (sebbo2002) [inline README.md]: I can't merge this if you rename the package in here. Sorry.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
-  "name": "ical-generator",
+  "name": "ical-generator-edtex",
+  "version": "v0.1.0",
```
Comment (sebbo2002) [inline package.json]: Version is added during the CI process, so please remove this line.

**Comment by sebbo2002** [comment]
Closed due to inactivity…

**Pull request closed without merging**